chapter05/qli/ex2.pl

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+
+
+chomp( my @inputs = <STDIN> );
+
+print "1234567890"x5, "\n" ;
```

Comment:
Dunno @alexdeas , we could argue over 'nice' vs 'cool' (:smile: vs :sunglasses: ?)

String repetition vs sequence operator, a loop and modulus maths..seems simpler this way?